server/planning/module.py

```diff
@@ -79,7 +81,14 @@ def init_planning(app: SuperdeskAsyncApp):
 module = Module(
     "planning",
     init=init_planning,
-    endpoints=[planning_locks_endpoint, planning_endpoint_group, events_endpoints_group, planning_download_endpoint],
+    endpoints=[
```

Comment:
I think these are playing up with the Web API for the app itself. The ContentAPI is meant to be run as a separate App to the Web API.
E2E tests are failing (which they're currently passing in async branch), and various behave tests are failing as well. Please have a look, and maybe need to run a separate test environment for the Content API

Reply:
Only the resources themselves are needed in the App config (so the publish into it works). The endpoints aren't needed

Reply:
but if we remove the endpoints here, we can't used it

Reply:
afaik, we need to define the endpoint group in the module config, right?

Reply:
We need to define these endpoint config in the ContenteAPI planning config, not the one for the main Web API app

Reply:
Got it, so I need to create a separate module for the planning`content_api` right ?